workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/Assembly.cs

```diff
@@ -19,11 +18,11 @@ public static class Assembly
                 "upload",
                 config.AssemblyName,
                 "--project_name",
-                config.ProjectName,
+                projectName,
```

Comment:
why is this not part of the config anymore?

Reply:
Hmmm this was a tricky one. We automatically grab the project name on load, so not entirely sure storing it in the scriptable object makes a _ton_ of sense as its not really "state". 

I can put it back in the config and copy it over whenever we load if you prefer?

Reply:
I have a mild preference to have it in the assembly config as you do need a project name to upload an assembly
but will leave it up to you.

Reply:
Moved it back into each of the configs